app/models/participant_profile.rb

```diff
@@ -3,6 +3,10 @@
 class ParticipantProfile < ApplicationRecord
   has_paper_trail
   belongs_to :teacher_profile, touch: true
+
+  # TODO: Back-fill, so that every profile has a schedule
+  belongs_to :schedule, optional: true, touch: true
```

Comment:
why the touch?

Reply:
I reckon if the schedule the participant is on changes, we would like to tell the lead providers about it in participant api. Especially if we end up letting IC's or participants change the schedule. 

Although... I guess maybe we want to only add that one after we populate the profiles with reasonable defaults to avoid 40k updated records 😂